server/update.sh

```diff
@@ -26,6 +26,9 @@ git reset --hard
 # Get the new version
 git pull
 
+# Store current SHA in a file
+git rev-parse HEAD > timestamp.txt
```

Comment:
See above. It still isn't timestamp, but commit hash. Maybe ```commit``` will be better filename?

Reply:
I give up. Renamed to commit.txt